luda-editor/client/src/pages/sequence_edit_page/components/wysiwyg_editor/mod.rs

```diff
@@ -114,51 +191,8 @@ impl Component for WysiwygEditor<'_> {
                         set_dragging.set(None);
                     }
                     Event::KeyDown { event } => {
-                        let moving_with = MovingWith::try_from(event.code);
-                        if let (Some(graphic_index), Ok(moving_with)) =
-                            (*editing_graphic_index, moving_with)
-                        {
-                            if cut_editor_focused {
-                                let set_new_dragging = || {
-                                    set_dragging.set(Some(Dragging::Mover {
-                                        context: MoverDraggingContext {
-                                            start_global_xy: Xy::zero(),
-                                            end_global_xy: Xy::zero(),
-                                            container_wh: wh,
-                                            moving_with,
-                                        },
-                                    }));
-                                };
-                                let mutate_dragging = || {
-                                    let mut delta_xy = moving_with.delta_xy();
-                                    if !namui::keyboard::shift_press() {
-                                        delta_xy *= 10.0;
-                                    }
-                                    set_dragging.mutate(move |dragging| {
-                                        if let Some(Dragging::Mover { context }) = dragging {
-                                            context.end_global_xy =
-                                                context.end_global_xy + delta_xy;
-                                        }
-                                    });
-                                };
-                                match *dragging {
-                                    Some(Dragging::Mover { context }) => {
-                                        if context.moving_with.key_changed(event.code) {
-                                            update_sequence_with_mover(context, graphic_index);
-                                            set_new_dragging();
-                                        }
-                                        mutate_dragging();
-                                        event.stop_propagation();
-                                    }
-                                    None => {
-                                        set_new_dragging();
-                                        mutate_dragging();
-                                        event.stop_propagation();
-                                    }
-                                    _ => {}
-                                }
-                            }
-                        }
+                        let _ = handle_graphic_order_change_shortcut(&event)
+                            || handle_graphic_move_shortcut(&event);
```

Comment:
What the hell..... seriously?

Reply:
Just create 'on_key_down' closure and make a match statement there. don't make this way which is difficult to understand your intent.

Reply:
Nah, I have no better idea for this.

Reply:
I have better options.
I think option 2 looks fine.

## option 1. more literal than original
```rust
if handle_graphic_order_change_shortcut(&event) {
} else if handle_graphic_move_shortcut(&event) {
}
```

## option 2. more kind option than option 1
```rust
Event::KeyDown { event } => 'handle_key_down: {
    let handle_graphic_order_change_shortcut_executed =
        handle_graphic_order_change_shortcut(&event);
    if handle_graphic_order_change_shortcut_executed {
        break 'handle_key_down;
    }

    let handle_graphic_move_shortcut_executed =
        handle_graphic_move_shortcut(&event);
    if handle_graphic_move_shortcut_executed {
        break 'handle_key_down;
    }
}
```